### PR TITLE
v0.77.0 - Update to node support to 12/14

### PR DESCRIPTION
**Pull request by godber**
This PR upgrades our CI to use node 14 as the primary node version and treats node 12 as the "old" compatibility release.  Node 10 is no longer tested at this point.

refs: #2704

### CODE REVIEW

**Comment by godber** [comment]
Also note my first commit was some cleanup and spelling.  There were duplicate keys that didn't make any sense.

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 dist: bionic
 os: linux
 language: node_js
-node_js: '12.20'
+node_js: '14'
```
Comment (godber) [inline .travis.yml:5]: Note, I only ever use `node_js: '14'` ... is there a minor version I should attach to these?  It was unclear what minor versions were acceptable.

Comment (peterdemartini) [reply .travis.yml:5]: Maybe `14.17`, will be our base version, maybe we should the base node 12 version to `12.22`

File: .travis.yml [hunk 2]
```diff
@@ -53,29 +53,52 @@ jobs:
...
       if: branch = master AND type IN (pull_request, cron) AND fork = false
       script:
         - export SERVICE_HEAP_OPTS="-Xms768m -Xmx768m"
         - yarn --silent --cwd e2e test
 
+    - name: 'Unit Test Suite (node 14)'
+      node_js: '14'
+      # run only on pull-requests or cron
+      if: branch = master AND type IN (pull_request, cron)
+      script: yarn --silent test --suite unit-b
```
Comment (godber) [inline .travis.yml:80]: I don't understand the rolls of suites `unit-a` and `unit-b`.  Basically what I've done here is moved the duplicated the node 12 tests with node 14 tests.  Should some of the old node 12 tests with ES be eliminated?

**Review by peterdemartini — APPROVED** [review]
LGTM

**Pull request merged**